stdlib/public/SwiftShims/HeapObject.h

```diff
@@ -180,6 +197,15 @@ static_assert(alignof(HeapObject) == alignof(void*),
   (__swift_uintptr_t) SWIFT_ABI_DEFAULT_OBJC_RESERVED_BITS_MASK
 #define _swift_abi_ObjCReservedLowBits                                         \
   (unsigned) SWIFT_ABI_DEFAULT_OBJC_NUM_RESERVED_LOW_BITS
+
+#if defined(__LP64__) || defined(__LLP64__)
```

Comment:
Nitpick: I think we've been using `__POINTER_WIDTH__ == 64` elsewhere in the runtime.

Reply:
Ah, thanks! I tried to see what was being used but didn't find that. LP64 is also used in a few places but for more platform-specific purposes, so this is definitely better.